root/static/scripts/common/MB/Control/Menu.js

```diff
@@ -24,16 +24,16 @@ MB.Control.HeaderMenu = function () {
     self.timeout = null;
     self.timeout_msecs = 200;
 
-    $('#header-menu > div > ul > li').bind('mouseenter.mb', function (event) {
+    $('.header ul.menu > li').bind('mouseenter.mb', function (event) {
         if (self.timeout) {
             clearTimeout(self.timeout);
-            $('#header-menu ul li ul').css('left', '-10000px');
+            $('.header ul.menu li ul').css('left', '-10000px');
         }
 
         $(this).children('ul').css('left', 'auto');
```

Comment:
I think I'll leave direction as it is right now. Might be something worth looking into though.